pkg/roachprod/vm/gce/gcloud.go

```diff
@@ -400,6 +400,30 @@ func (p *Provider) GetHostErrorVMs(
 	return hostErrorVMs, nil
 }
 
+// GetVMSpecs returns a json list of VM specs
```

Comment:
Nit: "...json list of VM specs, provided by GCE [1].

[1] https://cloud.google.com/sdk/gcloud/reference/compute/instances/describe"